aicsimageio/writers/ome_zarr_writer.py

```diff
@@ -105,6 +106,13 @@ def build_ome(
         }
         return omero
 
+    @staticmethod
+    def build_chunk_dims(
```

Comment:
I think this could be private

Reply:
resolved in [cdaa92e](https://github.com/AllenCellModeling/aicsimageio/pull/519/commits/cdaa92e40801f76f16fb7ccb4bef2e6f532eb54a)